pom.xml

```diff
@@ -416,7 +416,7 @@
         <version.org.jboss.mod_cluster>1.4.3.Final</version.org.jboss.mod_cluster>
         <version.org.jboss.narayana>5.12.0.Final</version.org.jboss.narayana>
         <version.org.jboss.openjdk-orb>8.1.5.Final</version.org.jboss.openjdk-orb>
-        <version.org.jboss.resteasy>3.15.1.Final</version.org.jboss.resteasy>
+        <version.org.jboss.resteasy>4.7.0.Beta1</version.org.jboss.resteasy>
```

Comment:
@jamezp Have we had any discussions yet about accepting a Beta component into WildFly?

Do we have a roadmap to indicate when this component will be going Final?

Reply:
@darranl The plan was the end of this month. I'm on PTO on the 1st and 2nd of July though and the 5th is a holiday here in the US. However I would say by the week of the 5th. I'll plan for it next week and look at open issues.